src/client/kotlin/tech/thatgravyboat/skyblockapi/api/area/SlayerAPI.kt

```diff
@@ -1,86 +0,0 @@
-package tech.thatgravyboat.skyblockapi.api.area
```

Comment:
skyblock api is a non breaking api so existing apis MUST be kept for a certain about of time